upgrade/network/chn_enable.md

```diff
@@ -0,0 +1,856 @@
+   ```
+
+### Create system backups
+
+**Recommend** copying and storing all data in ${BACKUPDIR} off-system in a version control repository is highly recommended.
+
+1. (`ncn-m001#`) Change to the backup directory.
+
+   ```bash
+   cd ${BACKUPDIR}
+   ```
+
+2. (`ncn-m001#`) Backup running system `SLS` data.
+
+   ```bash
+   curl -k -H "Authorization: Bearer ${TOKEN}" https://api-gw-service-nmn.local/apis/sls/v1/dumpstate | jq -S . > sls_input_file.json
+   ```
+
+3. (`ncn-m001#`) Backup running system customizations data.
+
+   ```bash
+   kubectl -n loftsman get secret site-init -o json | jq -r '.data."customizations.yaml"' | base64 -d > customizations.yaml
+   ```
+
+4. (`ncn-m001#`) Backup running system MetalLB `configmap` data.
+
+   ```bash
+   kubectl get cm -n metallb-system metallb -o yaml | egrep -v 'creationTimestamp:|resourceVersion:|uid:' > metallb.yaml
+   ```
+
+5. (`ncn-m001#`) Backup running system manifest data.
+
+   ```bash
+   kubectl get cm -n loftsman loftsman-platform -o jsonpath='{.data.manifest\.yaml}' > manifest.yaml
+   ```
+
+6. (`ncn-m001#`) Backup running system NCN Personalization data.
+
+   ```bash
+   cray cfs configurations describe ncn-personalization --format json | jq 'del(.lastUpdated) | del(.name)' > ncn-personalization.json
+   ```
+
+7. (`ncn-m001#`) Backup running system BSS data.
+
+   ```bash
+   curl -s -X GET -H "Authorization: Bearer ${TOKEN}" https://api-gw-service-nmn.local/apis/bss/boot/v1/bootparameters | jq . > bss-bootparameters.json
+   ```
+
+8. (`ncn-m001#`) Backup system databases (optional).
+
+   ```bash
+   TODO: DO WE NEED THIS?
+   ```
+
+## Update Phase
+
+**Recommend** copying and storing all data in ${UPDATEDIR} off-system in a version control repository is highly recommended.
+
+### Disable CFS for UAN
+
+1. (`ncn-m001#`) Disable CFS changes on UAN to prevent migration to the `CHN`.
+
+   ```bash
+   for xname in $(cray hsm state components list --role Application --subrole UAN --type Node --format json | jq -r .Components[].ID) ; do cray cfs components update --enabled false --format json $xname; done
+   ```
+
+### Update SLS
+
+1. (`ncn-m001#`) Move to the update directory.
+
+   ```bash
+   cd ${UPDATEDIR}
+   ```
+
+2. (`ncn-m001#`) Set the directory location for the `SLS` `CHN` script
+
+   ```bash
+   export SLS_CHN_DIR=/usr/share/doc/csm/upgrade/scripts/sls
+   ```
+
+3. (`ncn-m001#`) Add `CHN` to `SLS` data.
+
+   ```bash
+   ${SLS_CHN_DIR}/sls_can_to_chn.py --sls-input-file ${BACKUPDIR}/sls_input_file.json \
+      --customer-highspeed-network <CHN VLAN> <CHN IPv4 Subnet> \
+      --number-of-chn-edge-switches <number of edge switches> \
+      --sls-output-file ${UPDATEDIR}/sls_file_with_chn.json
+   ```
+
+   where:
+
+      - `<CHN VLAN>` is the "stub" `VLAN` for the `CHN`.  This is currently used only on the edge switches in access mode, not a trunk through the high speed network.
```

Comment:
Do we want to give them the value that we usually use for this vlan as an example?

Reply:
This has just lead to problems in the past so I'd prefer not to do this.  Plus defaults are baked into the python scripts and available on error or `--help`.